src/pages/Settings/GeneralSettingsSection.tsx

```diff
@@ -0,0 +1,42 @@
+import { useContext } from 'react'
+import { GlobalContext } from '../../App'
+import KeyValueInput from '../../components/Inputs/InlineLabelValueInput'
+import { Input } from '../../components/Inputs/Input'
+import ThemeSwitcher from '../../components/ThemeSwitcher'
+
+const GeneralSettingsSection = () => {
+  const {
+    settings: {
+      general: { walletLockTimeInMinutes }
+    },
+    updateSettings
+  } = useContext(GlobalContext)
+
+  return (
+    <>
+      <KeyValueInput
+        label="Lock time"
+        description="Duration in minutes after which the wallet will be locked automatically."
```

Comment:
I think this is a bit misleading. We should clarify that the wallet will only lock if it is inactive/idle for X minutes, otherwise we risk people disabling it completely if they think that it will ALWAYS lock after X minutes. How about:

> Duration in minutes after which an idle wallet will lock automatically.